backend/app/tests/api/routes/test_evaluation.py

```
@@ -792,3 +827,216 @@ def test_get_evaluation_run_trace_info_completed(
+        """Test deleting non-existent dataset returns 404."""
+        response = client.delete(
+            "/api/v1/evaluations/datasets/99999",
+            headers=user_api_key_header,
+        )
+
+        assert response.status_code == 404
+        response_data = response.json()
+        error_str = response_data.get(
+            "detail", response_data.get("error", str(response_data))
+        )
+        assert "not found" in error_str.lower()
```

Ruff (0.14.13)

No newline at end of file

Add trailing newline

(W292)